lib/paypal_express/application.rb

```diff
@@ -113,6 +113,19 @@ def plugin(session = {})
   end
 end
 
+# curl -v http://127.0.0.1:9292/plugins/killbill-paypal-express/1.0/account_external_keys/somebody@example.com?kb_tenant_id=some_uuid
+get '/plugins/killbill-paypal-express/1.0/account_external_keys/:email', :provides => 'json' do
+  if ids = ::Killbill::PaypalExpress::PaypalExpressResponse.uniq.where(:payer_email => params[:email]).pluck(:kb_account_id)
+    kb_tenant_id = request.GET['kb_tenant_id']
+    kb_tenant = request.env['killbill_tenant']
+    kb_tenant_id ||= kb_tenant.id.to_s unless kb_tenant.nil?
+
+    plugin(session).get_external_keys_for_accounts(ids, kb_tenant_id).to_json
```

Comment:
Would it make sense to return the full accounts json instead, so we get everything (account ids, etc.)?